src/test/java/duke/parser/CommandParserTest.java

```diff
@@ -0,0 +1,66 @@
+package duke.parser;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import org.junit.jupiter.api.Test;
+
+import duke.commands.Command;
+
+public class CommandParserTest {
```

Comment:
Nice tests implemented. Perhaps for completion you could implement the rest of the tests as well.